### PR TITLE
Updated install instructions for Windows direct install for CAVE (beta)

### PR DESCRIPTION
**Pull request by tiffanycmeyer13**

### CODE REVIEW

**Comment by srcarter3** [comment]
Do we want to push this after our next release of the beta?
If that's the case, do we want to update the version number in this same PR as well (from `20.3.2-0.1` to `20.3.2-0.2`)?

**Comment by tiffanycmeyer13** [comment]
Yes, this is in anticipation of our release. I've starting to go through our to do checklist

File: docs/install/install-cave-beta-v20.md [hunk 1]
```diff
@@ -163,3 +234,6 @@ sudo yum remove awips2-*
 rm -rf /awips2/cave
 rm -rf ~/caveData
 ```
+
+
+[def]: ../images/E
```
Comment (srcarter3) [inline docs/install/install-cave-beta-v20.md]: I think this very last line might have come from my commit? but I don't know what it is or if it needs to be here at all?

Comment (tiffanycmeyer13) [reply docs/install/install-cave-beta-v20.md]: Okay, I've got it removed now

**Pull request merged**